src/brevitas/graph/equalize.py

```diff
@@ -74,13 +86,16 @@ class WeightBiasTuple:
 class Region:
     srcs: Tuple = field(default_factory=tuple)
     sinks: Tuple = field(default_factory=tuple)
+    acts: Tuple = field(default_factory=tuple)
```

Comment:
What is acts in region? Why do we need it?

Reply:
It's convenient (but not fundamental) for the following:

- When equalizing through scale_varying activations, these are added as srcs, and we need to filter them out before equalizing
- When equalizing through scale_varying activations, we cannot equalize if a region contains two of them, so we need to keep track of how many we have encoutered

Originally there were more restrictions and acts was more useful than now. In principle we should be able to drop it